ts/features/payments/home/utils/transactionHistoryHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { fetchTransactionsRequestWithExpBackoff } from "../../../../store/actions/wallet/transactions";
```

Comment:
This is an hook, so why don't name its file accordingly? Like `useTransactionHistory`, for example. What do you think?

Reply:
the idea behind that naming was that the file would include more utils, but why not? at the end of the day if and when we'll need it we can just rename